lib/Minz/ModelPdo.php

```diff
@@ -114,20 +127,43 @@ public function __construct($currentUser = null, $currentPdo = null) {
 			);
 	}
 
-	public function beginTransaction() {
+	/**
+	 * @return void
+	 */
+	public function beginTransaction()
+	{
```

Comment:
Please keep same curly-brackets alignment than the rest of the document

Reply:
ok, your not PSR complient ?

Reply:
See https://github.com/FreshRSS/FreshRSS/blob/edge/phpcs.xml
In general, please keep the convention used in a given file

Reply:
@Frenzie If you have a minute, could you please check why this not caught by PHPCS?

Reply:
At a glance it looks like we only demand a newline after an opening brace and I guess no one's ever tried to use this style before. Probably easy to work around, let me see if I can figure it out in roughly 5 minutes (since I should sleep soon :-) ).

Reply:
https://github.com/FreshRSS/FreshRSS/pull/4122